### DISCUSSION TITLE
Where is the place for general requirements related to use case standards?

### DISCUSSION THREAD

**Discussion by eckardg** [catenax-eV/catenax-ev.github.io#118] [General]
In the industry core we are asked to give guidance for other use cases when the industry core has to be applied. Currently we define this in our industry core standard, as a "standard for other standards". Is this the right place for such a requirement? Or should it rather be a white paper, a general guideline?

**Comment by ClosedSourcerer** [comment]
CatX standards should work like ISO, or IEEE when it comes to their internal dependency logic:

1. Parent standards do not force things upon child standards.
2. Instead child standards adopt (partially) parent standards because of the value, that they bring.

This is consistent with the approach, that child standards are supposed to reference parent standards, including the version number of said parent standards.

Exempli gratia:

1. `General Clothing v1.0.0` allows for the covering of ones head
2. `Hats v1.0.0` describes different options on how to cover ones head and defines distinct hat shapes as well as names and references `General Clothing v1.0.0`
3. `Church leadership v1.2.0` adopts the part of `Hats v1.0.0` that describes the hat shaped called mitra and makes it mandatory for the pope in certrain situations

Because `Church leadership v1.2.0` references specifically `Hats v1.0.0` even if `Hats v2.0.0` would make wearing a mitra mandatory for every person alive it would not affect `Church leadership v1.2.0` directly. Instead `Church leadership v1.2.0` would most likely maintain the old reference for as long as possible, or abandon `Hats vX.Y.Z` should `Hats v1.0.0` itself be retired at some point.

**Reply by ClosedSourcerer** [reply]
This is also consistent with the value-add approach. 
If something adds value, adopt it. 
If it does not add value, don´t adopt it.
If it would add value, but you are unable to see it: Visit your political commisar.

**Comment by ma3u** [comment]
How we can define the changes/upgrade in a standard to define the dependencies?

https://jschuetz88.github.io/cx-dependencies/

Should we also track external standards e.g.  Dataspace Protocol, Decentralized Claims Protocol, IDTA/AAS?
